--- BackupPC-2.1.0/configure.pl	2004-06-20 02:28:08.000000000 +0000
+++ BackupPC-2.1.0.new/configure.pl	2004-12-06 22:28:42.445528048 +0000
@@ -250,7 +250,7 @@
 my($name, $passwd, $Uid, $Gid);
 while ( 1 ) {
     $Conf{BackupPCUser} = prompt("--> BackupPC should run as user",
-                                 $Conf{BackupPCUser} || "backuppc",
+                                 $Conf{BackupPCUser} || "builder",
                                  "username");
     ($name, $passwd, $Uid, $Gid) = getpwnam($Conf{BackupPCUser});
     last if ( $name ne "" );
